code/datums/craft/recipes/alchemy.dm

```diff
@@ -83,189 +93,195 @@
 	name = "Large Beaker"
 	result = /obj/item/reagent_containers/glass/beaker/large
 	steps = list(
-		list(/obj/item/stack/ore/glass, 1),
+		list(/obj/item/stack/ore/glass, 2),
 		list(QUALITY_HEATING, 5, 40)
 	)
 
-/datum/craft_recipe/alchemy/sulfur
-	name = "Sulfur Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sulfur
+// Preloaded chem dispenser-tier chemical bottles start here
+
+/datum/craft_recipe/alchemy/acetone
+	name = "Acetone Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/acetone
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 1), //Leaching the sulfur form rocks
-		list(/obj/item/reagent_containers/food/snacks/egg , 1), //For good luck?
+		list(/obj/item/stack/ore, 2), // Lead acetate
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/tungsten
-	name = "Tungsten Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/tungsten
+/datum/craft_recipe/alchemy/aluminum
+	name = "Aluminum Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/aluminum
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore/coal, 1),
-		list(/obj/item/stack/ore, 1),
+		list(/obj/item/stack/ore, 2), // Getting it from ore
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/radium
-	name = "Radium Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/radium
+/datum/craft_recipe/alchemy/ammonia
+	name = "Ammonia Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/ammonia
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore/uranium, 1),
+		list(/obj/item/stack/ore, 2), // Getting it from ore
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/sacid
-	name = "Sulphuric Acid Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sacid
+/datum/craft_recipe/alchemy/carbon
+	name = "Carbon Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/carbon
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 1), //Leaching the sulfur form rocks
-		list(/obj/item/reagent_containers/food/snacks/egg , 1), //For good luck?
-		list(/obj/item/reagent_containers/food/snacks/meat/xenomeat , 1), //Used for base, has pacid in
+		list(/obj/item/reagent_containers/food/snacks/meat, 3), // Any meat will do
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/sacid
-	name = "Sulphuric Acid Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sacid
+/datum/craft_recipe/alchemy/copper
+	name = "Copper Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/copper
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore/glass, 1),
+		list(/obj/item/stack/cable_coil, 30), // Cables have copper!
```

Comment:
hunters have exstreamly limited coils, using 30u of copper to get 60 well nice shouldnt replie on something they cant get more of

Reply:
This is balanced around life in the colony, if hunters and outsiders want to get more they can always tear apart machines here and there or open technical lockers, after all they have shared claims on Scrap Haven with the prospies
Also, once more, this PR removes the Alchemy perk from hunters, they already have the means to produce painkillers, antibiotics and chems with easy, I'm trying to gate alchemy behind crayon spells and commitment
This is also the reason the alchemy reagents no longer require tatonka milk but instead use normal milk